### PR TITLE
Replace pytest by unittest framework in docker lib

### PR DESCRIPTION
**Pull request by chris-verclytte**

### CODE REVIEW

File: .pylintrc [hunk 1]
```diff
@@ -138,7 +138,8 @@ disable=print-statement,
         xreadlines-attribute,
         deprecated-sys-function,
         exception-escape,
-        comprehension-escape
+        comprehension-escape,
+        bad-continuation
```
Comment (Olivier-Daire) [inline .pylintrc:142]: I am not sure about adding `bad-continuation` here, most of the time it is just a matter of formatting, and it helps keeping the code more readable

Comment (chris-verclytte) [reply .pylintrc:142]: Yep but it is definitely putting a false positive in our code with black behaviour and they don't agree to change the way it is working in black (see the comment description for link to both discussion on pylint and black repositories). The only workaround would be to put a local comment for disabling it. If you prefer, I can go with this way. Let me know.

Comment (Olivier-Daire) [reply .pylintrc:142]: Yes, if it's an edge case if think it might be worth putting a comment and not disabling it for the whole test module
